### PR TITLE
test(e2e): add "receive tokens" modal snapshot test

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

Add "receive tokens" modal snapshot test. The modal will be covered with additional tests in the next PRs.

### CODE REVIEW

File: e2e/receive-tokens-modal.spec.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { testWithII } from '@dfinity/internet-identity-playwright';
+import { MODALS_VIEWPORT_WIDTH } from './utils/constants/e2e.constants';
+import { HomepageLoggedIn } from './utils/pages/homepage.page';
+
+const RECEIVE_TOKENS_MODAL_VIEWPORT_HEIGHT = 900;
+
+const ICRC_ADDRESS_LABEL_SELECTOR = `[data-tid="${RECEIVE_TOKENS_MODAL_ICRC_SECTION}"] >> [data-tid="${RECEIVE_TOKENS_MODAL_ADDRESS_LABEL}"]`;
+const ICP_ADDRESS_LABEL_SELECTOR = `[data-tid="${RECEIVE_TOKENS_MODAL_ICP_SECTION}"] >> [data-tid="${RECEIVE_TOKENS_MODAL_ADDRESS_LABEL}"]`;
+const ETH_ADDRESS_LABEL_SELECTOR = `[data-tid="${RECEIVE_TOKENS_MODAL_ETH_SECTION}"] >> [data-tid="${RECEIVE_TOKENS_MODAL_ADDRESS_LABEL}"]`;
+const BTC_ADDRESS_LABEL_SELECTOR = `[data-tid="${RECEIVE_TOKENS_MODAL_BTC_SECTION}"] >> [data-tid="${RECEIVE_TOKENS_MODAL_ADDRESS_LABEL}"]`;
```
[the remark this answers is not in the document]
Comment (AntonioVentilii-DFINITY) [reply e2e/receive-tokens-modal.spec.ts:19]: and obviusly, use throughtout all the tests

File: e2e/about-how-modal.spec.ts
[the remark this answers is not in the document]
Comment (DenysKarmazynDFINITY) [reply e2e/about-how-modal.spec.ts:1]: Done, moved to a separate PR

File: e2e/about-what-modal.spec.ts
[the remark this answers is not in the document]
Comment (DenysKarmazynDFINITY) [reply e2e/about-what-modal.spec.ts:1]: Done, moved to a separate PR

File: e2e/receive-tokens-modal.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (DenysKarmazynDFINITY) [reply e2e/receive-tokens-modal.spec.ts:19]: sounds good

**Pull request merged**